environment.yaml

```diff
@@ -24,6 +24,8 @@ dependencies:
     - transformers==4.19.2
     - torchmetrics==0.6.0
     - kornia==0.6
+    - accelerate==0.12.0
+    - git+https://github.com/crowsonkb/k-diffusion.git@master
     - -e git+https://github.com/CompVis/taming-transformers.git@master#egg=taming-transformers
     - -e git+https://github.com/openai/CLIP.git@main#egg=clip
```

Comment:
@hwharrison Does this CLIP line need a change due to k-diffusion also including CLIP?

```
The conflict is caused by:
    The user requested clip 1.0 (from git+https://github.com/openai/CLIP.git@main#egg=clip)
    k-diffusion 0.0.1 depends on clip 1.0 (from git+https://github.com/openai/CLIP)

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```